dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/internal/LibIndexCache.java

```diff
@@ -66,7 +66,7 @@ protected void setAdaptableModuleFactory(AdaptableModuleFactory adaptableFactory
         this.adaptableFactory = adaptableFactory;
     }
 
-    public File getLibrary(Map.Entry<String, String> LibIndexEntry) {
+    private File getLibrary(Map.Entry<String, String> LibIndexEntry) {
```

Comment:
This and some other files seem unrelated to your changes.  Can you merge or rebase from integration?